src/me/mrCookieSlime/Slimefun/listeners/ItemListener.java

```diff
@@ -211,6 +211,8 @@ public void onRightClick(ItemUseEvent e) {
 		final Player p = e.getPlayer();
 		ItemStack item = e.getItem();
 
+		if (slimefunItem == null) return;
```

Comment:
This would prevent the Guide from opening and the debug fish from working, just remove it.
I already fixed it via a commit.